ospfd/ospf_asbr.c

```diff
@@ -53,8 +53,8 @@ void ospf_external_route_remove(struct ospf *ospf, struct prefix_ipv4 *p)
 	rn = route_node_lookup(ospf->old_external_route, (struct prefix *)p);
 	if (rn)
 		if ((or = rn->info)) {
-			zlog_info("Route[%s/%d]: external path deleted",
-				  inet_ntoa(p->prefix), p->prefixlen);
+			zlog_info("Route[%pI4/%d]: external path deleted",
```

Comment:
Isn't this just %pFX  if we are going to fix it... fix it?

Reply:
Addressed.